frontend/src/components/pages/ViewDonations.tsx

```diff
@@ -25,6 +28,16 @@ const ViewDonations = (): React.ReactElement => {
     Date | DateObject | DateObject[] | null
   >(new Date());
   const [test, setTest] = useState<any>(0);

```

Comment:
lol this test thing is so weird. Why do we need this? I also tried removing it and running it, but the left right icons do not work if I do that.

Reply:
DateObject is a custom object from the library we use. I'm assuming it's not triggering a state update and that's why we need it. I think this workaround is fine for now since trying to fix this will take way longer than required. I can create an issue for this so that we can circle back to this if you'd like.